src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import BookList from './components/BookList';
+import Categories from './components/Categories';
+import Navigation from './components/Navigation';
+
+function App() {
+  return (
+    <>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<BookList />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+    </>
```

Comment:
- [ ] in the App component you are calling the booklist directly which is rendering a list of books, according to the requirements however you should call a component "books" and from the books component you should call the "bookform" and the "booklist"